src/renderer/store/settings.store.ts

```diff
@@ -281,6 +282,7 @@ const initialState: SettingsState = {
     general: {
         accent: 'rgb(53, 116, 252)',
         defaultFullPlaylist: true,
+        externalLinks: false,
```

Comment:
Default this to `true`. I'd say it's better to have users have to opt-out here rather than opt-in since people may need look through every setting.